source/context_menus/_context_menu_stats.js

```diff
@@ -0,0 +1,58 @@
+const { PermissionFlagBits, InteractionContextType } = require('discord.js');
```

Comment:
```suggestion
const { InteractionContextType } = require('discord.js');
```
PermissionFlagBits is unused